kafka/pykafka/consumer.py

```diff
@@ -47,9 +199,63 @@ class OwnedPartition(object):
     Used to keep track of offsets and the internal message queue.
     """
 
-    def __init__(self, partition):
+    def __init__(self, partition, consumer):
```

Comment:
Adding `consumer` makes this a nice little reference cycle, which Python's GC can handle, but is a bit confusing.  It may be better to have `consumer_group=None` and `fetch_message_max_bytes=1` (or whatever) to avoid the cycle and make it a straighter object graph.